### ISSUE TITLE
cross origin is not optional

### ISSUE THREAD

**Issue by hdramd** [matheusdavidson/angular-cropperjs#1]

**Comment by matheusdavidson** [comment]
Hey @hdramd, i'll put that as an option, will update in minutes.

**Comment by hdramd** [comment]
I still have the same problem.

<angular-cropper  [cropperOptions]="config" [imageUrl]="imageUrl"></angular-cropper>

  config = {
    checkCrossOrigin: false
  };

**Comment by matheusdavidson** [comment]
I didn't finish yet, found i problem.

**Comment by matheusdavidson** [comment]
Fixed, update to 0.1.5.

Run `npm update` and use:
```js
config = {
    checkCrossOrigin: false
};
```

```html
<angular-cropper [cropperOptions]="config" [imageUrl]="imageUrl">
```

**Comment by matheusdavidson** [comment]
Take a look at this issue:
https://github.com/fengyuanchen/cropper/issues/119

Looks like you have some limitations when you don't have crossOrigin, have a look.

Did the fix work for you?

**Comment by hdramd** [comment]
its work...thanks alot

**Comment by matheusdavidson** [comment]
fixed in b34b696 b86afb9
